recommenders/evaluation/tf_evaluation.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import tensorflow as tf
+
+
+def rmse(rating_true, rating_pred):
+    """Root Mean Square Error
```

Comment:
Small add: maybe you can mention that this is RMSE `using Tensorflow` just to distinguish it from the other rmse that we have in the repo

Reply:
Done.